src/risedevtool/src/task/kafka_service.rs

```diff
@@ -12,108 +12,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::env;
-use std::path::{Path, PathBuf};
-use std::process::Command;
+use super::docker_service::{DockerService, DockerServiceConfig};
+use crate::KafkaConfig;
 
-use anyhow::{anyhow, Result};
-
-use super::{ExecuteContext, Task};
-use crate::{KafkaConfig, KafkaGen};
-
-pub struct KafkaService {
-    config: KafkaConfig,
-}
-
-impl KafkaService {
-    pub fn new(config: KafkaConfig) -> Result<Self> {
-        Ok(Self { config })
+impl DockerServiceConfig for KafkaConfig {
+    fn id(&self) -> String {
+        self.id.clone()
     }
 
-    fn kafka_path(&self) -> Result<PathBuf> {
-        let prefix_bin = env::var("PREFIX_BIN")?;
-        Ok(Path::new(&prefix_bin)
-            .join("kafka")
-            .join("bin")
-            .join("kafka-server-start.sh"))
+    fn is_user_managed(&self) -> bool {
+        self.user_managed
     }
 
-    fn kafka(&self) -> Result<Command> {
-        Ok(Command::new(self.kafka_path()?))
+    fn image(&self) -> String {
+        "confluentinc/cp-kafka:7.6.1".to_owned()
```

Comment:
Is it possible to have a place define all the versions of such docker images? Also is it possible to clean old versions of these images if we later upgrade the version?

Reply:
> Is it possible to have a place define all the versions of such docker images?

You reminded me that the image name should be customizable through `risedev.yml`.

> Also is it possible to clean old versions of these images if we later upgrade the version?

We cannot tell if the image is pulled by RiseDev or the user, so let's just leave it to the user.